package.json

```diff
@@ -17,37 +17,63 @@
 		"watch": "NODE_ENV=development webpack --progress --watch --config webpack.js"
 	},
 	"dependencies": {
+		"@babel/eslint-parser": "^7.14.7",
 		"@juliushaertl/vue-richtext": "^0.3.3",
 		"@nextcloud/auth": "^1.3.0",
 		"@nextcloud/axios": "^1.6.0",
 		"@nextcloud/browser-storage": "^0.1.1",
 		"@nextcloud/capabilities": "^1.0.4",
 		"@nextcloud/dialogs": "^3.1.2",
+		"@nextcloud/eslint-plugin": "^2.0.0",
 		"@nextcloud/event-bus": "^2.0.0",
 		"@nextcloud/initial-state": "^1.2.0",
 		"@nextcloud/l10n": "^1.4.1",
 		"@nextcloud/moment": "^1.1.1",
 		"@nextcloud/paths": "^2.0.0",
 		"@nextcloud/router": "^2.0.0",
-		"@nextcloud/vue": "^4.0.2",
+		"@nextcloud/vue": "^4.0.3",
 		"@nextcloud/vue-dashboard": "^2.0.1",
+		"@tensorflow-models/body-pix": "^2.2.0",
+		"@tensorflow/tfjs": "^3.7.0",
+		"@types/leaflet": "^1.7.4",
 		"attachmediastream": "^2.1.0",
+		"babel-eslint": "^10.1.0",
+		"babel-loader": "^8.2.2",
 		"color.js": "^1.2.0",
 		"crypto-js": "^4.0.0",
+		"css-loader": "^4.3.0",
 		"debounce": "^1.2.1",
 		"emoji-regex": "^9.2.2",
 		"escape-html": "^1.0.3",
+		"eslint": "^7.30.0",
```

Comment:
You don't need to include eslint and webpack, these are implicitly imported from the existing dev dependencies `@nextcloud/webpack-vue-config` and others. Not sure if this happened on its own while you updated your branch with master ?

You only need to add the deps needed for video processing.

Reply:
Thanks for the advice!
The dependencies got accidentally while trying to get the dev installation to work. Should have used `--no-save`.
I will commit a fix.